.changeset/curvy-taxis-float.md

```diff
@@ -0,0 +1,12 @@
+---
+"@thirdweb-dev/unity-js-bridge": patch
+"@thirdweb-dev/react-native": patch
+"@thirdweb-dev/react-core": patch
+"@thirdweb-dev/wallets": patch
+---
+
+[Wallets]
+
+- Updates base wallet classes names
+- Makes InjectedWallet params optional
+- Removes connector storage constructor param from Injected and MetaMask wallets
```

Comment:
if it's only a `wallets` change then let's only make a changeset for `wallets`

Reply:
well, it's a wallets change but it affects the other packages since we export an updated base wallet class which `react-native`, `react-core` and `unity` use

Reply:
But they depend on wallets so will get bumped because of that already